### ISSUE TITLE
no such module 'GCDWebServer'

### ISSUE THREAD

**Issue by artworker** [tapthaker/FBSimulatorClient#1]
How can i build this?

i'm newbe 

![2015-11-16_195407](https://cloud.githubusercontent.com/assets/1291903/11180159/e9fe8fc8-8c9b-11e5-9235-0f7d1de02a62.png)

I can't build with this message

`No such module 'GCDWebServer`

**Comment by abhishek-maniyar** [comment]
did you run `pod install` ?? 
For more details please refer [Getting started with CocoaPods](https://guides.cocoapods.org/using/getting-started.html)

**Comment by artworker** [comment]
yes I did.

I tried multiple times.

but no luck.

my xcode version is 7.x is this make problum?

**Comment by tapthaker** [comment]
Yes. It can be a problem because of swift 2 / 2.1 changes. Will get back to you in sometime.
